docs/gitbook/architecture/topology/considerations.adoc

```diff
@@ -0,0 +1,97 @@
+= Key Considerations
+
+This section describes the key considerations for blockchain topologies,
+which are:
+
+* <<_privacy>>
+* <<_security>>
+* <<_modularity>>
+* <<_scalability>>
+
+
+== Privacy
+
+For a single blockchain, all parties with access to the blockchain have
+access to all of its link:/key-concepts/state.adoc[application state].
+This means that privacy is not a core feature of this topology; privacy
+would only be somewhat achievable via obfuscation or anonymization.
+
+With multiple blockchains,
+link:/key-concepts/permission_model.adoc[fine-grained access controls],
+and the ability to be link:chain_joins.adoc#slices[selective about what
+data is shared], true privacy can be achieved using the Interbit
+platform.
+
+For example, if a user has a private blockchain containing all of their
+personal data, including (but not limited to) mailing address, health
+records, financial records, family and business relationships, that user
+can decide to share only their mailing address with the blockchain
+owned by another user, but could share specific health records with
+their insurance provider's blockchain, all without disclosing any other
+information.
+
+Privacy verification is also a key feature: each blockchain and its
+associated smart contract demonstrate which data is shared, when, and to
+which other party. All changes to data sharing, access, and the logic
+managing those capabilities is stored immutably on the associated
+blockchains.
+
+A best practice for privacy is to begin with the most granular privacy
+controls required, and then relax them where necessary. By contrast,
+beginning with no privacy concerns and then adding them later is
+significantly more challenging and error-prone. A thorough privacy
+analysis at the start of your project would reveal the appropriate
+blockchain topology to use.
+
+
+== Scalability
+
+"Traditional" blockchains suffer from one of two main scalability
+issues:
+
+- low transaction rate imposed by proof-of-work consensus
+- unified state managed by a single point in the network
+
+Proof-of-work consensus involves each node in the network performing
+CPU-intensive calculations that take a notable amount of time to
```

Comment:
I suggest something like "computation-intensive" instead of "CPU-intensive", because it isn't always a CPU involved - it is often GPUs now.